lib/mastodon/cli/accounts.rb

```diff
@@ -166,6 +166,7 @@ def modify(username)
       user.disabled = true if options[:disable]
       user.approved = true if options[:approve]
       user.otp_required_for_login = false if options[:disable_2fa]
+      user.webauthn_credentials = [] if options[:disable_2fa]
```

Comment:
Thanks for the contribution!

I think it might make more sense to use `User#disable_two_factor!` in order to preserve consistency and avoid code duplication. Although looking at that code, one complication is that this method directly saves the `user`.

Reply:
Thank you for your comment! 👍  I will revise the code and push to this PR.

Reply:
I have submitted a new commit 🚀